assets/pod-checkpointer-operator/daemonset.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: pod-checkpointer
+  name: pod-checkpointer
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: pod-checkpointer
+  template:
+    metadata:
+      labels:
+        k8s-app: pod-checkpointer
+      annotations:
+        checkpointer.alpha.coreos.com/checkpoint: "true"
+    spec:
+      containers:
+        - name: pod-checkpointer
+          command:
+            - /checkpoint
+            - --kubeconfig=/etc/kubernetes/kubeconfig
+            - --container-runtime-endpoint=unix:///var/run/crio/crio.sock
```

Comment:
does the checkpointer talk directly to the runtime?

Reply:
it does.